pkg/config/config.go

```diff
@@ -84,6 +84,7 @@ func (c *Config) fillDefaults() error {
 		URL:             "https://localhost:6443",
 		Port:            6443,
 	}
+	c.ApiServer.AuditLog.Profile = "Default"
```

Comment:
Reference https://github.com/openshift/api/blob/master/config/v1/types_apiserver.go#L81 ?

Reply:
Ah, good catch. It does effectively duplicate the test. I'll remove the e2e test and leave the unit.

Reply:
I think this is the answer to the unit test vs e2e test comment? :)